.github/workflows/release.yaml

```diff
@@ -323,13 +323,21 @@ jobs:
       - uses: actions/setup-node@v2.5.1
         with:
           node-version: "16"
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.17"
+      - run: go install sigs.k8s.io/bom/cmd/bom@v0.2.0
```

Comment:
very minor but one thing I did for ArgoCD repo was parametrizing the versions as env vars to make it easier to update in future.